### PR TITLE
Do not recalculate enhanced metric setting each time.

### PR DESCRIPTION
**Pull request by purple4reina**
### What does this PR do?

Sets the value for enhanced metrics as a global instead of recalculating it each time.

### Types of Changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [ ] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [ ] This PR passes the integration tests (ask a Datadog member to run the tests)

### CODE REVIEW

File: datadog_lambda/extension.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import logging
-from os import path
+import os
```
Comment (duncanista) [inline datadog_lambda/extension.py:2]: why importing the whole`os` instead of `path` from `os`?

File: datadog_lambda/metric.py [hunk 2]
```diff
@@ -32,6 +32,10 @@
     flush_in_thread = os.environ.get("DD_FLUSH_IN_THREAD", "").lower() == "true"
     lambda_stats = ThreadStatsWriter(flush_in_thread)
 
+enhanced_metrics_enabled = (
```
Comment (duncanista) [inline datadog_lambda/metric.py:35]: why a tuple?

File: datadog_lambda/extension.py (hunk 1 above)
Comment (purple4reina) [reply datadog_lambda/extension.py:2]: habit. I'm just used to seeing `os.path.exists` not `path.exists`.

File: datadog_lambda/metric.py (hunk 2 above)
Comment (purple4reina) [reply datadog_lambda/metric.py:35]: It's not a tuple. Tuples have a comma in them. Without the comma it's just a way to put something on multiple lines.

Comment (duncanista) [reply datadog_lambda/metric.py:35]: Oop, tysm :til:

**Review by duncanista — APPROVED** [review]

**Pull request merged**